core/shared/src/main/scala/scalaz/zio/ZSchedule.scala

```diff
@@ -671,33 +707,36 @@ private[zio] trait Schedule_Functions extends Serializable {
    * </pre>
    */
   final def spaced(interval: Duration): Schedule[Any, Int] =
-    forever.delayed(_ + interval)
+    forever.delayed(_ + interval.relative)
 
   /**
    * A schedule that always recurs, increasing delays by summing the
    * preceding two delays (similar to the fibonacci sequence). Returns the
    * current duration between recurrences.
    */
   final def fibonacci(one: Duration): Schedule[Any, Duration] =
-    delayed(unfold[(Duration, Duration)]((Duration.Zero, one)) {
-      case (a1, a2) => (a2, a1 + a2)
-    }.map(_._1))
+    delayed[Any, Any](
```

Comment:
Is the `[Any, Any]` type annotation required? It looks like it was not required in the old version.

Reply:
I'm trying running this locally over dotty 0.13.0-RC1 and without type annotation an error is thrown.
```
no implicit argument of type Schedule_Functions.this.ConformsR[Nothing] was found for parameter evidence$2 of method delayed in trait Schedule_Functions
```

Reply:
Dangit.